app/src/main/kotlin/com/metrolist/music/ui/screens/settings/integrations/MatrixSettings.kt

```
@@ -0,0 +1,369 @@
+@Composable
+fun MatrixSettings(
+    navController: NavController,
+) {
+    val (matrixRPC, onMatrixRPCChange) = rememberPreference(EnableMatrixRPCKey, false)
+    var accountsJson by rememberPreference(MatrixAccountsKey, "[]")
+    var statusFormat by rememberPreference(MatrixStatusFormatKey, "")
+
+    val accounts = remember(accountsJson) {
+        try {
+            matrixJsonSerializer.decodeFromString<List<MatrixAccount>>(accountsJson)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)